torchdynamo/symbolic_convert.py

```diff
@@ -170,6 +171,14 @@ def wrap_local(self, name, value):
                 state=TracingSupported.YES,
                 guards={Guard(name, GuardSource.LOCAL, GuardBuilder.TYPE_MATCH)},
             )
+        elif isinstance(value, Real):
+            self.graphargs.append(LocalArg(name))
+            self.create_graph_input(name)
+            return ConstantVariable(
```

Comment:
ConstantVariable => TensorVariable

Reply:
Either way works.  If you want to try adding a new one that could be useful later on.